network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/primary_heroguts.html

```diff
@@ -3,7 +3,18 @@
 <div id="hero" class="mb-md-3">
   <div class="banner">
     {% if page.banner %}
-    {% image page.banner original alt="" %}
+    <picture>
+      {% image page.banner fill-4032x1152 as image_xl %}
+      {% image page.banner fill-2400x686 as image_lg %}
+      {% image page.banner fill-1984x567 as image_md %}
+      {% image page.banner fill-1536x439 as image_sm %}
+      <source media="(min-width: 1200px)" srcset="{{ image_xl.url }}"  alt="">
+      <source media="(min-width: 992px)" srcset="{{ image_lg.url }}"  alt="">
+      <source media="(min-width: 768px)" srcset="{{ image_md.url }}"  alt="">
+      <source media="(min-width: 576px)" srcset="{{ image_sm.url }}"  alt="">
+      {# Fallback Image #}
+      {% image page.banner fill-1008x288 alt="" %}
```

Comment:
Can you add alt text for accessibility here (for `source` and `{% image %}`)? :)